ui/test/nuxeo-document-layout.test.js

```diff
@@ -110,6 +123,70 @@ suite('<nuxeo-document-layout>', () => {
     expect(event.detail.layout).to.equal(documentLayout.layout);
   });
 
+  test('Should reload when document changes', async () => {
+    documentLayout = await buildLayout();
+    expect(documentLayout.element).to.not.be.null;
```

Comment:
We can try to align and define some "internal convention" since this kind of assertion can be done in different ways.
This tests contains both approaches:
`expect(documentLayout.element).to.not.be.null;` and `expect(element).to.exist;`

Reply:
IMHO, I'd say that we should always be the most specific as possible. So if we know exactly what the value should be or should not be, I think we should use it (be it `undefined` or `null`). In this case, we use both `undefined` and `null` (which is inconsistent), so I think we can go with `to.exist`. (already addressed in one of the comments above)